index.html

```diff
@@ -2,9 +2,113 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Domein.shop</title>
+    <link rel="stylesheet" href="styles/styles.css">
 </head>
 <body>
-
+<header>
+    <div class="header-items">
+        <h1>Domein.shop</h1>
+        <p>Jouw domeinnaam registreren in slechts 1 minuut!</p>
+        <form onsubmit="JS">
+            <label for="domein">
+                <input class="header-input" type="text" id="domein" name="domein"
+                       placeholder="Is jouw domein nog vrij?"/>
+            </label>
+            <button class="header-button" type="button">
+                Check beschikbaarheid
+            </button>
+        </form>
+    </div>
+</header>
+<main>
+    <section class="testimonial-section">
+        <h2>Wat zeggen klanten over ons</h2>
+        <article class="testimonial-article">
+            <img class="quote-icon" src="/assets/icons/quote-left.png" alt="quote-icon"/>
```

Comment:
Het icon is bij mij niet zichtbaar
```suggestion
            <img class="quote-icon" src="assets/icons/quote-left.png" alt="quote-icon"/>
```